bundle/jsky.app.ot/src/main/java/jsky/app/ot/gemini/editor/targetComponent/EdCompTargetList.java

```diff
@@ -105,9 +105,10 @@ public void textBoxKeyPress(final TextBoxWidget tbwe) {
         _agsPub.subscribe((obs, oldOptions, newOptions) -> updateGuiding());
 
         BagsManager.addBagsStateListener((key, oldStatus, newStatus) -> {
-            if (key.equals(getContextObservation().getNodeKey()))
+            if (Optional.ofNullable(getContextObservation()).map(co -> key.equals(co.getNodeKey())).orElse(false)) {
```

Comment:
I don't have a strong opinion on this. Our stuff is probably better than Java's so I'm not inclined to get rid of it, especially if we're phasing out OCS.